streamer/src/recvmmsg.rs

```diff
@@ -1,23 +1,21 @@
 //! The `recvmmsg` module provides recvmmsg() API implementation
 
 pub use solana_perf::packet::NUM_RCVMMSGS;
-use {
-    crate::packet::{Meta, Packet},
-    std::{cmp, io, net::UdpSocket},
-};
 #[cfg(target_os = "linux")]
 use {
+    crate::msghdr::create_msghdr,
     itertools::izip,
-    libc::{
-        iovec, mmsghdr, msghdr, sockaddr_storage, socklen_t, AF_INET, AF_INET6, MSG_WAITFORONE,
-    },
+    libc::{iovec, mmsghdr, sockaddr_storage, socklen_t, AF_INET, AF_INET6, MSG_WAITFORONE},
     std::{
         mem::{self, MaybeUninit},
         net::{Ipv4Addr, Ipv6Addr, SocketAddr, SocketAddrV4, SocketAddrV6},
         os::unix::io::AsRawFd,
-        ptr,
     },
 };
+use {
+    crate::packet::{Meta, Packet},
+    std::{cmp, io, net::UdpSocket},
+};
```

Comment:
nit: superfluous reorder?

Reply:
Not my choice: rustfmt does this sometimes, and I'm not sure why. If I change it back, it'll just do this again

Reply:
i'm sure it will move it back next rust version bump